bot/cogs/wolfram.py

```diff
@@ -0,0 +1,46 @@
+import discord
+from io import BytesIO
+from urllib import parse
+from discord.ext import commands
+from bot.settings import APPID
+
+
+class Wolfram(commands.Cog):
+    """Command for Wolfram search."""
+    def __init__(self, bot):
+        self.bot = bot
+        self.query = "http://api.wolframalpha.com/v2/{request}?{data}"
+        
+
+    @commands.command(brief="Takes in a wolfram search and displays the result", usage=".wolfram [query]")
+    async def wolfram(self, ctx, *, query):
+        """Wolfram command, takes in a search and gives the result."""
+        url_str = parse.urlencode({
+            "i": query,
+            "appid": APPID,
+        })
+
+        query_final = self.query.format(request="simple", data=url_str)
+
+        async with ctx.channel.typing():
+            async with self.bot.session.get(query_final) as response:
+                assert response.status == 200, response.raise_for_status()
+                image_bytes = await response.read()
+
+                image_file = discord.File(BytesIO(image_bytes), filename="image.png")
+                image_url = "attachment://image.png"
+
+                message = ""
+                footer = "View original for a bigger picture."
+                color = discord.Colour.orange()
+                                
+                final_emb = discord.Embed(title=message, color=color)
+                final_emb.set_image(url=image_url)
+                final_emb.set_footer(text=footer)
+ 
+                await ctx.send(embed=final_emb, file=image_file)
+
+def setup(bot):
+    """Sets up the Wolfram cog."""
+    bot.add_cog(Wolfram(bot))
+    
```

Comment:
on my end there are two blank lines at the end of the file, where there should only be one. I do not see it on github, so it just might be a bug for me. 
![image](https://user-images.githubusercontent.com/39353605/97614625-69493300-19d7-11eb-80ca-7c9ddeaa9c81.png)

Reply:
actually i do have 2 blank lines at the end, maybe patryk or ren fixed it on github earlier though so it isnt showing on github